### DISCUSSION TITLE
6 xch disappear

### DISCUSSION THREAD

**Discussion by nuistzj2014** [Chia-Network/chia-blockchain#5316] [Support]
I did not send any transactions,but searched two txs using 'chia wallet get_transactions' ,  show below:

Transaction 3d114216f0fe46f15ce4b4f5bcfbff2d0b9b696f9adb898fe941e0eb******
Status: Confirmed
Amount: 0.5 xch
To address: xch1f6va7a5d7hmnr8qsxlshvvkztwgnzuft7c2vs2fna69cuuy48vhs*******
Created at: 2021-05-16 08:22:03

Transaction 393e13421175dde5ec91fb4318f889defc2c2b6dac9e8b1147ffe769******
Status: Confirmed
Amount: 0.01 xch
To address: xch1puavgacap7jyhkkjw5z02u07htszcq9yp4nusy3cvpjextgt8a7qs*******
Created at: 2021-05-16 08:30:33

Even more surprising，the two transactions send 0.51 xch, but I lost 6 xch ,here is my address record in chiaexplorer:

05/15/2021, 06:57:01 AM | 🌱 Farmer reward | 0x3408833fdc97b869da504e2b986ff14d74ca14bd4834de20a12f2d19ffd**** | 1.75 XCH | $2,353.98 | 281,884 | 287,146
-- | -- | -- | -- | -- | -- | --

05/15/2021, 06:57:01 AM | 🌱 Farmer reward | 0xcfa4d7c0938ecc4afec197ffadc6c2278b9a5b52d3c68234d9bff2ec833***** | 0.25 XCH | $336.28 | 281,884 | 287,160
-- | -- | -- | -- | -- | -- | --

05/15/2021, 02:21:35 AM | 🌱 Farmer reward | 0x007f634de8210923a5950a6a3b0785df4820bf4e1c2adac00392e59940e****2 | 0.26 XCH | $349.73 | 280,928 | 287,160
-- | -- | -- | -- | -- | -- | --

05/15/2021, 02:21:35 AM | 🌱 Farmer reward | 0xb928b76b5a9d1c1fd957c3e05d09810aa30d1fdabb20d46587084f2*******2 | 1.75 XCH | $2,353.98 | 280,928 | 287,160
-- | -- | -- | -- | -- | -- | --

05/14/2021, 06:28:30 PM | 🌱 Farmer reward | 0xbe81f6a81a6bfe84688d4073eb238f8c118dd20aacabc01f6c2ca0********93c | 0.25 XCH | $336.28 | 279,204 | 287,160
-- | -- | -- | -- | -- | -- | --

05/14/2021, 06:28:30 PM | 🌱 Farmer reward | 0x4d21b9a7fb85ba9c0801096af209d854dec22a93cf4c5af4d090********38058 | 1.75 XCH | $2,353.98 | 279,204 | 287,146
-- | -- | -- | -- | -- | -- | --

**Comment by Sabcoll** [comment]
Are you maybe on HPool?

**Reply by nuistzj2014** [reply]
No, I farmed on my own!

**Reply by nuistzj2014** [reply]
The  all transactions' xch cannot match the total xch I farmed. I've checked it many times.

**Comment by kruiescody** [comment]
Can you guys help me to get started. How much will I need to get a chia rig set up

**Comment by ceednaan** [comment]
Did you receive your xch ? My 4 xch disappear like yours :(

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
